packages/e2eTesting/integration/checkbox.spec.ts

```diff
@@ -0,0 +1,75 @@
+/// <reference types="cypress" />
+context('Actions', () => {
+  before(() => {
+	  cy.visit('localhost:8080/#/components/Checkbox')
+  })
+	it('Validate checkbox is checked', ()=>{
```

Comment:
The formatting of this file seems a little off. It seems as though Prettier is not setup properly to run automatically.

Reply:
Yea.. my prettier doesnt work since pretty much the beginning of my internship i didnt remember this was that bad though 🤔